### DISCUSSION TITLE
Commercial use

### DISCUSSION THREAD

**Discussion by Hiromi-Kusutani** [vdemichev/DiaNN#1257] [Q&A]
We are CRO in Japan. We provide testing services to our clients.
Is it possible to use the database for testing servise?

**Comment by vdemichev** [comment]
Hi Hiromi,

Please see https://github.com/vdemichev/DiaNN/releases/tag/1.9.2, you will need a license from us, please get in touch!

Best,
Vadim

**Comment by Hiromi-Kusutani** [comment]
Hello Vadim,

Thank you for your reply.
We are CRO in Japan. We would like to use your database for our testing service for our clients.
We look into  the URL and it says

  *   Our spin-off Aptila Biotech<https://aptila.bio/> is now preparing an enterprise version of DIA-NN for Industry.
We would like to know the Terms & conditions for commercial use. Should we contact Aptila Biotech directly?

Your comments on this would be highly appreciated.

Best,
Hiromi
****************************************************
Hiromi Kusutani
Intellectual Property
Axcelead, Inc.
Atago Green Hills MORI Tower 21F
2-5-1 Atago, Minato-ku, Tokyo,105-6221
Tel: +81-80-7005-5186
E-mail: ***@***.******@***.***>
****************************************************
